app/utils/storage-utils.spec.ts

```diff
@@ -5,7 +5,7 @@ describe("StorageUtils", () => {
     let startAndEndRegex: RegExp;
 
     beforeEach(() => {
-        multipleCharRegex = (StorageUtils as any)._regexUnderscoresAndMultipleDashes;
+        multipleCharRegex = (StorageUtils as any)._regexInvalidCharacters;
```

Comment:
is this being used?

Reply:
It's used in the 4th `it` block. I'm not sure exactly why though, maybe just to validate that the regex is what we expect, without testing the StorageUtils class? I can remove if it's unnecessary, I think it was a direct copy from the portal tests.